src/content/docs/ja/recipes/renovate.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Renovate
+description: RenovateのBotを設定する
+---
+
+[Renovate](https://docs.renovatebot.com/) は、広く使われている依存関係の更新の自動化ツールです。
+
+## biome.json
+
+Renovate は、`biome.json` ファイル内の `$schema` のバージョンを最新の状態に更新する [preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) を提供しています。
```

Comment:
```suggestion
Renovate は、`biome.json` ファイル内の `$schema` のバージョンを最新の状態に更新する [shared preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) を提供しています。
```

Can be written as same as [renovate.mdx](https://github.com/biomejs/website/blob/main/src/content/docs/recipes/renovate.mdx?plain=1#L10) for consistency?

Reply:
OK, I applied your suggestion!